boards/dwm1001/include/periph_conf.h

```diff
@@ -1,4 +1,4 @@
-/*
+su/*
```

Comment:
left-over :)

Reply:
that was probably meant for a different window :-)